pkgs/os-specific/linux/criu/default.nix

```diff
@@ -1,6 +1,8 @@
 { stdenv, lib, fetchFromGitHub, fetchpatch, protobuf, protobufc, asciidoc, iptables
 , xmlto, docbook_xsl, libpaper, libnl, libcap, libnet, pkg-config, iproute2
-, which, python3, makeWrapper, docbook_xml_dtd_45, perl, nftables, libbsd }:
+, which, python3, makeWrapper, docbook_xml_dtd_45, perl, nftables, libbsd
+, buildPackages, buildPlatform, targetPlatform
```

Comment:
Top-level buildPlatform and targetPlatform are deprecated, use  `stdenv.xPlatform`

Reply:
fixed it for you

Reply:
fixed it for you

Reply:
Thanks!